webpack/assets/javascripts/react_app/components/DiffView/DiffRadioButtons.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { ButtonGroup, Button } from 'patternfly-react';
+import { translate } from '../../common/I18n';
+
+const btnClass = (stateView, btnView) =>
+  classNames('diff-button', { active: stateView === btnView });
+
+const radioButton = (stateView, btnView, changeState) => (
+  <Button
+    className={btnClass(stateView, btnView)}
+    id={`${btnView}-btn`}
+    onClick={() => changeState(btnView)}
+    bsStyle={stateView === btnView ? 'primary' : 'default'}
+  >
+    {translate(btnView)}
+  </Button>
+);
+
+const DiffRadioButtons = ({ stateView, changeState }) => (
+  <div id="diff-radio-buttons">
+    <ButtonGroup>
+      {radioButton(stateView, 'split', changeState)}
+      {radioButton(stateView, 'unified', changeState)}
```

Comment:
I think you should define the translation here, e.g.``` __('split')``` ,this is important for two reasons:
1.```translate``` is not a keyword that our string extraction code catches /cc @amirfefer IMHO docs needs to reflect this.
2. this approach will not actually extract the string, just translate it, hence it will never actually show it as translated.